usr/lib/live-installer/installer.py

```diff
@@ -416,9 +416,17 @@ def finish_install(self, setup):
             if(line.startswith("XKBMODEL=")):
                 newconsolefh.write("XKBMODEL=\"%s\"\n" % setup.keyboard_model)
             elif(line.startswith("XKBLAYOUT=")):
+                if setup.keyboard_layout in ('am', 'ara', 'ben', 'bd', 'bg', 'bt', 'by', 'deva', 'ge',
+                      'gh', 'gr', 'guj', 'guru', 'il', 'in', 'ir', 'iku',
+                      'kan', 'kh', 'kz', 'la', 'lao', 'lk', 'mk', 'mm', 'mn',
+                      'mv', 'mal', 'ori', 'pk', 'ru', 'scc', 'sy', 'syr',
+                      'tel', 'th', 'tj', 'tam', 'ua', 'uz'):
+                    setup.keyboard_layout = 'us,%s' % self.setup.keyboard_layout
```

Comment:
But why add the US layout?

Reply:
@kernc  - https://github.com/linuxmint/lmde-3-cinnamon-beta/issues/10